src/foundry/foundry.js/pixi/filters/abstractBaseMaskFilter.d.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * This class defines an interface for masked custom filters
+ */
+declare abstract class AbstractBaseMaskFilter extends PIXI.Filter {
+  /**
+   * The default vertex shader used by all instances of AbstractBaseMaskFilter
+   */
+  static vertexShader: string;
+
+  /**
+   * The fragment shader which renders this filter.
+   * A subclass of AbstractBaseMaskFilter must implement the fragmentShader(channel) static field.
+   */
+  static fragmentShader: ((channel: string) => string) | null;
+
+  /**
+   * A factory method for creating the filter using its defined default values
+   * @param defaultUniforms - Initial uniforms provided to the filter
+   *                          (default: `{}`)
+   * @param channel         - A color channel to target for masking.
+   *                          (default: `r`)
+   */
+  static create(
+    defaultUniforms?: ConstructorParameters<typeof PIXI.Filter>[2],
+    channel?: string
```

Comment:
I think this probably can only be 'r', 'g', 'b' (and maybe 'a' if alpha channels are also supported)